tools/packaging/kernel/build-kernel.sh

```diff
@@ -415,10 +416,37 @@ build_kernel() {
 	popd >>/dev/null
 }
 
+build_kernel_headers() {
+	echo "### BUILD KERNEL HEADERS"
+	local kernel_path=${1:-}
+	[ -n "${kernel_path}" ] || die "kernel_path not provided"
+	[ -d "${kernel_path}" ] || die "path to kernel does not exist, use ${script_name} setup"
+	[ -n "${arch_target}" ] || arch_target="$(uname -m)"
+	arch_target=$(arch_to_kernel "${arch_target}")
+	pushd "${kernel_path}" >>/dev/null
+
+	if [ "$linux_headers" == "deb" ]; then
```

Comment:
CI is failing with `/home/runner/work/kata-containers/kata-containers/tools/packaging/kernel/build-kernel.sh: line 428: linux_headers: unbound variable`.

Reply:
This should be fixed now.